pkgs/by-name/te/tetrio-plus/package.nix

```diff
@@ -14,17 +14,17 @@
   nodejs,
   asar,
 
-  tetrio-src,
-  tetrio-version,
+  tetrio-desktop,
 }:
 
 let
   version = "0.27.2";
+  rev = "electron-v${version}-tetrio-v${lib.versions.major tetrio-desktop.version}";
 
   src = fetchFromGitLab {
     owner = "UniQMG";
     repo = "tetrio-plus";
-    rev = "electron-v${version}-tetrio-v${lib.versions.major tetrio-version}";
+    inherit rev;
```

Comment:
Why move rev?

Reply:
Used in changelog, wanted to reduce duplication, can revert.